scripts/local-release

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+# The purpose of this script is to publish artifacts to ~/local-repo
+
+if [ ! -f gradle.properties ]; then
+  echo 'Could not find gradle.properties. Run this command from the root of the project.'
+  exit 1
+fi
+
+if [ ! -d "$HOME" ]; then
+  echo 'Cannot perform local release, $HOME is not set to a valid directory.'
+  exit 1
+fi
+
+LOCAL_REPO="${HOME}/local-repo"
+
+if [ ! -d $LOCAL_REPO ]; then
+  mkdir $LOCAL_REPO
+fi
+
+VERSION=$(awk 'BEGIN { FS = "=" }; $1 == "version" { print $2 }' gradle.properties | awk '{ print $1 }')
+echo "Publishing parseq $VERSION to ${LOCAL_REPO}..."
+
+./gradlew -i --console=plain -Dmaven.repo.local=$LOCAL_REPO -Prelease publishReleasePublicationToMavenLocal
```

Comment:
What is "maven.repo.local" system property doing? Can you document it? (I assume what it does but let's be explicit).

nit: you can use "publishToMavenLocal" instead of "publishReleasePublicationToMavenLocal". The former will publish all publications (you have only one "release" anyway).

Reply:
It overrides the location of Maven local to be `~/local-repo` so that the artifacts will be published there instead. I'll add a comment explaining this.

As for the second comment, I could do that but I figured it would be best to be explicit.